libscript/src/binary.mlc

```diff
@@ -203,7 +203,7 @@ Tags: Binary
 syntax EmptyData is expression
 	"the" "empty" "data"
 begin
-	MCBinaryEvalEmpty(output)
+	MCDataEvalEmpty(output)
 end syntax
```

Comment:
It's probably not important, but wouldn't it be better to keep the prefix consistent and correct the name in the module-binary.cpp file to be MCBinaryEvalEmpty?